### DISCUSSION TITLE
No se me agregó a repositorio de mi grupo

### DISCUSSION THREAD

**Discussion by maurolaurenzi** [dds-jv/foro#115] [Consultas generales] [answered]
Hola, soy del grupo 21 de los viernes a la noche. Me registré en enma pero no se me agregó al repositorio de mi grupo. Quería saber si pueden agregarme. Gracias.

**Comment by gastonprieto** [comment]
Buenas,

Lo que veo es que se te envio una invitación para pertener a la organización, falta que la aceptes, deberías tener un mail de github.com y sino creo que entrando a la pagina logueado te aparece una notificación

**Comment by walterbarreiro** [comment]
Buenas, soy del grupo 6 de los jueves a la mañana y me pasó lo mismo salvo que yo si acepté la invitación al grupo pero aun así no me aparece el repositorio. Saludos

**Comment by gastonprieto** [comment]
@walterbarreiro listo. Mas allá de eso trata de no usar el mismo issue porque sino no podemos trackear a quien le respondimos y a quien no.

**Comment by walterbarreiro** [comment]
Ok, lo tendré en cuenta para la próxima. Gracias!

**Comment by gastonprieto** [comment]
@maurolaurenzi pudiste acceder?

**Comment by maurolaurenzi** [comment, marked as answer]
Hola. Sí, pude! Gracias!
